index.html

```diff
@@ -13,19 +14,26 @@
     <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&display=swap" rel="stylesheet">
     <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&family=Roboto:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
     <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,200;0,300;0,400;0,700;1,100;1,200;1,300;1,400;1,700&family=Roboto:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
-    
+    <link rel="stylesheet" href="queries.css">
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <header >
         <nav>
-        <a href="">John Doe</a>
-         <img src="Images/532-5328945_menu-bar-icon-white-clipart-png-download-menu.png" alt=""> 
+        <a href="" class="john">John Doe</a>
+        <div class="menu">
+            <a href="#Portfolio" class="icon">Portfolio</a>
+            <a href="#About-me" class="icon">About</a>
+            <a href="#Contact" class="icon">Contact</a>
+            <a href="" class="icon message"><img src="Images/message-16.png" alt=""></a>
+         <img src="Images/532-5328945_menu-bar-icon-white-clipart-png-download-menu.png" alt="" class="dropdown">
+        </div> 
         </nav>
```

Comment:
- [ ] For the desktop version, please increase the padding of this element as suggested by the previous reviewer. 👍